src/main/resources/application.yml

```diff
@@ -17,8 +17,8 @@ spring:
   mail:
     host: ${MAIL_HOST:smtp.gmail.com}
     port: ${MAIL_PORT:587}
-    username: ${MAIL_USERNAME}
-    password: ${MAIL_PASSWORD}
+    username: ${MAIL_USERNAME:sellerpanel2@gmail.com}
```

Comment:
@vishalgupta921 this is public repo, we should not have credentials exposed in yml.